guidelines/templates/resources/details.md

```diff
@@ -0,0 +1,18 @@
+---
+title: {Document title}
+type: Details
+---
+
+>**NOTE:** Blockquotes in this document provide instructions. Remove them from the final document.
+>
+>This is a template for the **Details** document type that provides more in-depth, technical details about a given Kyma component. This document should include, among other things, a detailed explanation of the application lifecycle that describes how the resource is created and what other resources are created, how it is updated, how it is removed, and what each operation means from the technical point of view.  
+>
+>  In the document:
+> * Provide an introductory sentence that explains what the document is about.
+> * Divide the document into sections to make it more reader-friendly.
+> * Use headings to name sections. Use H2 (##) for the main sections and H3 (###) for the sub-sections. Do not use headings smaller than H3 as they do not display well.
+> * Provide any relevant links to the further documentation.
```

Comment:
Good job. I have only one comment to the template that relates to the point where you mention the links - since we cannot cross-reference internal docs that point may be misleading. Maybe you can rephrase it somehow and mention: `Provide references to the further documentation. Do not use cross-references between documents but mention the related document title instead. Put the document name in bold.`

Reply:
Discussed.